plugins/modules/registration_command.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) Evgeni Golov
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: registration_command
+version_added: 4.0.0
+short_description: Manage Registration Command
+description:
+  - Manage Registration Command
+author:
+  - "Evgeni Golov (@evgeni)"
+options:
+  activation_keys:
+    description:
+    - Activation keys for subscription-manager client, required for CentOS and Red Hat
+      Enterprise Linux.
+    - Required only if host group has no activation keys.
+    required: false
```

Comment:
Is `required: false` the default when writing these?

Reply:
yeah